SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -2,172 +2,33 @@
 
 This document outlines the process for releasing a version of the GDK for Unreal and all associated projects.
 
-> This process will need to be reviewed once we hit beta.
-
 ## Terminology
-* **Release version** is the version of the SpatialOS GDK for Unreal that you are releasing by performing the steps in this document.
-* **Previous version** is the latest version of the SpatialOS GDK for Unreal that is currently released to customers. You can find out what this version is [here](https://github.com/spatialos/UnrealGDK/releases).
-* `<GameRoot>` - The directory that contains your project’s .uproject file and Source folder.
-* `<ProjectRoot>` - The directory that contains your `<GameRoot>` directory.
-* `<YourProject>` - The name of your project and .uproject file (for example, `\<GameRoot>\YourProject.uproject`).
-
-## Validation pre-requisites
-
-The following entry criteria must be met before you start the validation steps:
-
-### Ensure Xbox DLLs exist
```

Comment:
Is this no longer required?

Reply:
It's not. See [WRK-1676](https://improbableio.atlassian.net/browse/WRK-1676?focusedCommentId=274961).